bootstrap-joining-demo/marathon-api-docker/build.sbt

```diff
@@ -1,7 +1,21 @@
 import com.typesafe.sbt.packager.docker._
 
+name := "bootstrap-joining-demo-marathon-api-docker"
+
+version := "0.1.0"
+
+scalaVersion := "2.12.4"
+
 enablePlugins(JavaServerAppPackaging)
 
 version := "1.0"
 
 dockerUsername := sys.env.get("DOCKER_USER")
+
+val akkaManagementVersion = "0.10.0"
+
+libraryDependencies ++= Vector(
+  "com.lightbend.akka.management" %% "akka-management-cluster-bootstrap" % akkaManagementVersion,
+  "com.lightbend.akka.management" %% "akka-management-cluster-http"      % akkaManagementVersion,
+  "com.lightbend.akka.discovery"  %% "akka-discovery-marathon-api"       % akkaManagementVersion
```

Comment:
Version management of those will become a pain... Please do depend on the root project.

Reply:
Yes, people should change then the version to the latest manually, and that's fine.